### PR TITLE
Fission output type blog post

### PR DESCRIPTION
**Pull request by gauravgahlot**
Signed-off-by: Gaurav Gahlot <gauravgahlot0107@gmail.com>

**What type of PR is this?**

/kind content

**Any specific area of the project related to this PR?**

/area blog

### CODE REVIEW

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:17]:
```suggestion
> * [Kubernetes Response Engine, Part 7 : Falcosidekick + Cloud Functions]({{< ref "/blog/falcosidekick-reponse-engine-part-7-cloud-functions" >}})
```

**Comment by Issif** [comment]
/hold

We can't release this post before next version of falcosidekick

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:38]:
```suggestion
There are various ways to provision a local Kubernetes cluster such as KinD, k3s, k0s, Minikube, etc. We are going to
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:24]:
```suggestion
In this blog post, we will cover using `Falcosidekick` and `Fission` to detect and delete a compromised pod in a Kubernetes cluster.
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:248]:
```suggestion
We create a `ServiceAccount` with rights to delete a Pod in any namespace, and we'll associate it to our function:
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (leogr) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:491]:
```suggestion
With this really simple example, we got another way to create a Response Engine with amazing pieces of software from the Open Source world.
```

**Review by leogr — COMMENTED** [review]
Just found some minor issues, otherwise looks good to me! :clap:

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:3]: Please update the date to a more recent one (after release of 2.24.0 of falcosidekick)

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:3]:
```suggestion
date: 2021-08-29
```

**Review by Issif — CHANGES_REQUESTED** [review]
please update the date of the post

**Review by Issif — APPROVED** [review]

**Review by leogr — APPROVED** [review]
LGTM

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:2]:
```suggestion
title: "Kubernetes Response Engine, Part 9: Falcosidekick + Fission"
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:5]:
```suggestion
slug: falcosidekick-reponse-engine-part-9-fission
```

File: content/en/blog/falcosidekick-reponse-engine-part-8-fission.md
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-8-fission.md:17]: Can you add the part 8 with Flux v2 please

**Review by Issif — CHANGES_REQUESTED** [review]

**Review by Issif — APPROVED** [review]

**Comment by gauravgahlot** [comment]
Needs a rebase after #492 is merged.

**Review by leogr — APPROVED** [review]

**Pull request merged**
